pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/thread/ThreadRuntime.java

```diff
@@ -179,7 +179,6 @@ public void start() throws Exception {
                 String.format("%s-%s",
                         FunctionCommon.getFullyQualifiedName(instanceConfig.getFunctionDetails()),
                         instanceConfig.getInstanceId()));
-        this.fnThread.setContextClassLoader(functionClassLoader);
```

Comment:
@jerrypeng 
why are we dropping this line ?

Reply:
When we start JavaInstanceRunnable, the context classloader needs to be set to instanceClassLoader, which is done by default, because the instanceClassLoader contains all the classes for the function instance / framework.  The functionClassLoader only contains classes that were submitted by the user via a JAR or a NAR.